### PR TITLE
Currency UI Script

### PR DESCRIPTION
**Pull request by FireRedNinja**

### CODE REVIEW

File: Roots and Froots/Assets/Scripts/UI/Currency.cs [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+    private void Start() {
+        transform.GetComponent<TextMeshProUGUI>();
+        SetCurrencyValue(0);
+    }
+    public void SetCurrencyValue(int value) {
+        string curr_name;
+        if (value == 1) {
+            curr_name = "groat";
+        } else {
+            curr_name = "groats";
```
Comment (Yasmojam) [inline Roots and Froots/Assets/Scripts/UI/Currency.cs:19]: nit: currency looks important can we have some tests?

Comment (FireRedNinja) [reply Roots and Froots/Assets/Scripts/UI/Currency.cs:19]: no. we need to push to prod now

Comment (Yasmojam) [reply Roots and Froots/Assets/Scripts/UI/Currency.cs:19]: fair enough

**Review by Yasmojam — APPROVED** [review]
LGTM

File: Roots and Froots/Assets/Scripts/UI/Currency.cs (hunk 1 above)
Comment (josh-richardson) [reply Roots and Froots/Assets/Scripts/UI/Currency.cs:19]: eat it then reet it mate, we don't need no tests

**Pull request merged**